packages/xdl/src/Project.js

```diff
@@ -39,6 +42,7 @@ import * as Android from './Android';
 import Api from './Api';
 import ApiV2 from './ApiV2';
 import Config from './Config';
+import createCompiler from './createCompiler';
```

Comment:
Maybe call the function & file createWebpackCompiler so it makes more sense in this context where everything else is not necessarily about webpack